package com.example.travelVer2.repository;

import org.springframework.data.domain.Page;
import org.springframework.data.domain.Pageable;
import org.springframework.data.jpa.repository.JpaRepository;

import com.example.travelVer2.entity.Reservation;
import com.example.travelVer2.entity.User;

public interface ReservationRepository extends JpaRepository<Reservation, Integer> {
	public Page<Reservation>findByUserOrderByCreatedAtDesc(User user,Pageable pageable);

}
